ime/app/src/main/java/com/anysoftkeyboard/keyboards/AnyKeyboard.java

```diff
@@ -832,7 +832,13 @@ public AnyKey(
 
             // if the shift-character is a symbol, we only show it if the SHIFT is pressed,
             // not if the shift is active.
-            mShiftCodesAlways = mShiftedCodes.length == 0 || Character.isLetter(mShiftedCodes[0]);
+            // Note: above doesn't work, it only shows the symbols if SHIFT is being held.
```

Comment:
I'm also confused about the comment above....

Anyhow, what are you fixing here?

Reply:
If a shifted keycode is a combining character it doesn't show when shift is pressed.

Reply:
Basically the code block that i changed was supposed to make it so that unless shift was purposefully pressed, only alphabet letters would be shown. This stops special symbols from showing. I don't really understand the functionality except on bottom bars where you can have an exclamation mark that changes into a question mark or something on shift press. But, this code doesn't even work.